content/source/docs/cloud/guides/recommended-practices/part3.1.html.md

```diff
@@ -13,15 +13,15 @@ Allow one individual (or a small group) in your engineering team to get familiar
 
 ## 1. Install Terraform
 
-[Follow the instructions here to install Terraform OSS](https://learn.hashicorp.com/terraform/getting-started/install).
+[Follow the instructions here to install Terraform OSS](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS).
```

Comment:
Do we want to add `?in=terraform/aws-get-started` in this link? User isn't looking for the AWS page in particular

Reply:
Mmm, good point. My eyes were crossing a bit by this point, I think. 

```suggestion
[Follow the instructions here to install Terraform OSS](https://learn.hashicorp.com/tutorials/terraform/install-cli?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS).
```